sql/src/main/java/org/apache/druid/sql/calcite/rel/DruidRel.java

```diff
@@ -95,6 +96,26 @@ public PlannerContext getPlannerContext()
     return plannerContext;
   }
 
+  /**
+   * Overridden to ensure that subclasses provide a proper implementation. The default implementation from
+   * {@link AbstractRelNode} does nothing and is not appropriate.
+   */
+  @Override
+  public RelWriter explainTerms(RelWriter pw)
+  {
+    throw new UnsupportedOperationException();
+  }
+
+  @Override
+  protected Object clone() throws CloneNotSupportedException
+  {
+    // RelNode implements Cloneable, but our class of rels is not cloned, so does not need to implement clone().
+    throw new UnsupportedOperationException();
```

Comment:
IntelliJ inspections is flagging it as it does not throw a CloneNotSupportedException